include/blogPost.php

```diff
@@ -0,0 +1,48 @@
+<?php
+function getArray(){
+    $post = array(
+        array(
+            'postid'=>0,
+            'title'=> "Getting To Know Me <br/>",
+            'body'=> "<div class = 'aboutMe'>
+                    Below are some more facts about me: <br/> <br/>
+                    I am from NY <br/>
+                     My favorite dessert is ice cream <br/>
+                     My least favorite forrd is chocolate <br/>
+                     I am scared of cats <br/>
+             My favorite Disney movie is mulan <br/><br/><br/>"
+         ),
+         array(
+             'postid'=>1,
+             'title'=> "What I am Studying <br/>",
+             'body'=> "<div class = 'aboutMe'>
+                     I am currently studying CogNeuro<br/><br/><br/>",
+          ));
+          return $post;
+}
+
+function getPost(){
+    $post = getArray();
+    for ($i = 0; $i<count($post); $i++){
+        $postId = $post[$i]['postid'];
+        $postTitle = $post[$i]['title'];
+        $postBody= $post[$i]['body'];
+      echo  "<a class = 'resume' href = 'viewPost.php?postid=". $postId."'>".$postTitle." </a><br/>";
+    }
+}
+
+
+
+function blog(){
```

Comment:
Sorry to keep saying the same thing, but this is another place where I don't think the function name properly communicates what the function does. Something like `echoBlogPostHtml` would be more informative.

Additionally, while what you're doing here will definitely work, it's normally only necessary to put code into a function it you think you'll be re-using it in multiple places. Since this prints all of the html for a blog post, it seems like you'll probably only us it in one place (viewPost.php). So I'd probably convert this to be a function that *gets* the info about a blog post and returns it (without echoing anything) and then echo the html in viewPost.php. a `getPost` function would be more re-usable than this one is right now.